### ISSUE TITLE
Add back drip functionality until customer.io ready for message switch

### ISSUE THREAD

**Issue by Timo614** [big-neon/bn-api#1453]
See https://github.com/big-neon/bn-api/pull/1450

Short: It was requested to add customer.io so we can switch to that for drip notifications so I handled it via webhooks on domain events. The problem is I removed the drips at that time but it's unlikely we'll be ready to turn on customer.io to handle the drip messaging for a little bit as we'll need to import all those domain events and then setup the workflows.

This approach is a bit more light handed and gives us a chance to swap over to customer.io for messaging in a more orderly fashion. 

Note: making issue so we have something to track for QA flow.

On the QA side we just need to confirm that drips still go out via the tooling. I already tested it locally and saw them going out.

**Comment by Tas4tari** [comment]
@nine-tari just to confirm, we have kept the drip notification exactly the same as to what it was previously. We did not add in any of the changes from the document that is on the issue in Asana: 
Document: https://docs.google.com/document/d/1cA3yXaY9pbigrUqmn-Y5YhCwgVFYgPLhANJhnOLJxYc/edit?ts=5d2fb623
Asana issue: https://app.asana.com/0/1131887530977355/1122751922498036

**Comment by Timo614** [comment]
@Tas4tari yeah it's the old drips here. The new drips are set to be handled via customer.io I believe once we're ready there.
